src/os/bluestore/bluefs_types.cc

```diff
@@ -200,18 +204,29 @@ void bluefs_fnode_t::generate_test_instances(list<bluefs_fnode_t*>& ls)
   ls.back()->size = 1048576;
   ls.back()->mtime = utime_t(123,45);
   ls.back()->extents.push_back(bluefs_extent_t(0, 1048576, 4096));
-  ls.back()->__unused__ = 1;
+  ls.back()->type = 0;
 }
 
 ostream& operator<<(ostream& out, const bluefs_fnode_t& file)
 {
-  return out << "file(ino " << file.ino
+  static constexpr std::string_view node_type_extra_data[] =  {
```

Comment:
IMO looks like an overkill - you're printing type name under "if(type == WAL_V2)" check anyway